packages/@uppy/audio/src/PermissionsScreen.tsx

```diff
@@ -12,9 +12,9 @@ export default (props: PermissionsScreenProps) => {
   return (
     <div className="uppy-Audio-permissons">
       <div className="uppy-Audio-permissonsIcon">{icon()}</div>
-      <h1 className="uppy-Audio-title">
+      <div className="uppy-Audio-title">
```

Comment:
I assume we want to keep the accessibility similar to `<h1>`, no?
```suggestion
      <div className="uppy-Audio-title" role="heading" aria-level="1">
```

Reply:
I thought about it, but felt like cargo-culting accessibility to me.

First of all, this is not actually a heading, this is just call to action text.

When I ran this with voiceover, browser's:

<img width="438" alt="image" src="https://github.com/user-attachments/assets/7ed3de1f-fef6-4126-a09d-c1a6eca9bb58">

is what voiceover was reading of course.

If the user clicks "Block" there, or if the site automatically opens Uppy's Audio/Camera as the default action, making this text a heading will actually confuse the screenreaders - screenreaders can read all headings on the page as an overview, and "Please allow access to your camera" will be one of the page headings - pretty sure unintentioned consequence.

___

One argument pro `role="heading" aria-level="1"` is that screenreaders change the tone of the voice when reading headings, which is what we'd want in this case.

Reply:
My point is - I'd say making it a heading in the first place was a mild accessibility mistake, we don't need to recreate that mistake with aria tags now.

Reply:
Maybe what we should do is to add `role=dialog` to the enclosing `<div>`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role, which seems to allow using heading that are separated from the rest of the document – or maybe we should use `<dialog>` element, but I expect that'd be a more involved change.

Reply:
Currently we have:
- for inline mode: `role={undefined} aria-modal={false}`
- for modal mode: `role="dialog" aria-modal={true}`

As I understand it, you are suggesting we make inline-uppy `role="dialog"` too.

I don't think we should do it, because the general expectation is that focus does wrap inside of the dialog (even in `aria-modal={false}` case [I think](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role#required_javascript_features)), and we don't wrap focus in inline mode.

Also - even if we made Uppy a dialog in inline mode, that wouldn't make "Please allow access to your camera" a heading 😄

Reply:
I'd say if something _is_ a heading in Uppy, it's the "Import from Audio" part

<img width="719" alt="image" src="https://github.com/user-attachments/assets/1b84cc30-85f3-457f-9269-1891d6cd916b">

Reply:
But also wouldn't go for making it an `<h1/>` for reasons mentioned above

Reply:
yeah ok that makes sense, it just feels "wrong" to have a `div.uppy-Audio-title` without any special role – maybe there's a tag out there (`<strong>`?), but you're right that stop using `<h1>` would be an improvement in any case.